app/web/template/certificate_provider_your_details.gohtml

```diff
@@ -1,13 +1,36 @@
 {{ template "page" . }}
 
-{{ define "pageTitle" }}{{ end }}
+{{ define "pageTitle" }}{{ tr .App "yourDetails" }}{{ end }}
 
 {{ define "main" }}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <p class="govuk-body">TODO, but proof it works:</p>
-      <p class="govuk-body">Donor is {{ .Lpa.You.FullName }}</p>
-      <p class="govuk-body">Certificate provider is {{ .Lpa.CertificateProvider.FullName }}</p>
+      <form novalidate method="post">
+        <div class="govuk-form-group">
+          <fieldset class="govuk-fieldset">
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+              <h1 class="govuk-fieldset__heading">{{ tr .App "yourDetails" }}</h1>
+            </legend>
+
+            <h2 class="govuk-heading-m">{{ tr .App "donor" }}</h2>
+            <p class="govuk-body">{{ .Lpa.You.FullName }}</p>
```

Comment:
I only put this in to show it worked, probably don't want donor info on the "your details" page, but also don't want to get too sidetracked on the design of a page that will surely change...

Reply:
I'll leave as is for now and we'll fix up when we get the real design for the page.